web-ui/src/main/resources/catalog/components/edit/validationreport/ValidationReportDirective.js

```diff
@@ -88,6 +88,14 @@
                     });
                   });
 
+                  // Move xsd rules to the end of the mandatory rules
+                  var xsdItem = _.find(scope.ruleTypes, function(r) { return r.id == 'xsd'; });
+                  if (xsdItem !== undefined) {
+                    scope.ruleTypes = _.without(scope.ruleTypes, xsdItem);
+
+                    scope.ruleTypes.push(xsdItem);
+                  }
+
```

Comment:
We know that XSD error are always first in the API, so maybe we could do:
```suggestion
                  if (scope.ruleTypes[0].id === 'xsd') {
                    scope.ruleTypes.push(scope.ruleTypes.splice(0, 1)[0]);
                  }
```
and then optional rules are appended.